s2spy/time.py

```diff
@@ -304,3 +296,237 @@ def show(self) -> pd.DataFrame:
         return self._label_targets(self.get_intervals()).applymap(
             self._interval_as_weeknr
         )
+
+
+class CustomCalendar(BaseCalendar):
+    """Build a calendar from sratch with basic construction elements."""
+
+    def __init__(self, anchor: Tuple[int, int] = (11, 30)):
```

Comment:
Let's not have a default value for the anchor. It's very arbitrary and users will always have to set it.

Reply:
Thanks for the suggestion. Corrected!